test/fixtures/section-heading-first-fail.html

```diff
@@ -0,0 +1,3 @@
+<section>
+    <p>Hello, world.</p>
+</section>
```

Comment:
Oh I see what you mean. We have two different fixtures here, one for `pass` which should contain passing examples, and another for `fail` which should contain failing examples. I was following Christine's previous work/examples.